GitOut/Features/Git/Log/GitRemoteViewModel.cs

```diff
@@ -0,0 +1,42 @@
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace GitOut.Features.Git.Log
+{
+    public class GitRemoteViewModel : INotifyPropertyChanged
+    {
+        private bool isSelected;
+
+        public GitRemoteViewModel(GitRemote model)
+        {
+            Name = model.Name;
+            Model = model;
+        }
+
+        public string Name { get; }
+        public bool IsSelected
+        {
+            get => isSelected;
+            set => SetProperty(ref isSelected, value);
+        }
+        public GitRemote Model { get; }
+
+        public event PropertyChangedEventHandler? PropertyChanged;
+
+        public static GitRemoteViewModel From(GitRemote model) => new GitRemoteViewModel(model)
+        {
+            IsSelected = true
+        };
```

Comment:
I think it would be better if isselected is set via the ctor to avoid the propertychanged event being fired

Reply:
Can do